### PR TITLE
Add loongarch64 examples

### PR DESCRIPTION
**Pull request by zhaixiaojuan**
Add loongarch64 support in examples.
Added test files: https://github.com/zhaixiaojuan/object-testfiles/commit/088cf625d14b776dee02b499838b0a3ce7b3fc8f

### CODE REVIEW

File: crates/examples/src/readobj/elf.rs [hunk 1]
```diff
@@ -2940,6 +2950,58 @@ static FLAGS_R_NDS32: &[Flag<u32>] = &flags!(
...
+    R_LARCH_ADD8,
+    R_LARCH_ADD16,
+    R_LARCH_ADD24,
+    R_LARCH_ADD32,
+    R_LARCH_ADD64,
+    R_LARCH_SUB8,
+    R_LARCH_SUB16,
+    R_LARCH_SUB24,
+    R_LARCH_SUB32,
+    R_LARCH_SUB64,
```
Comment (philipc) [inline crates/examples/src/readobj/elf.rs:3003]: Is this missing `R_LARCH_GNU_VTINHERIT` and `R_LARCH_GNU_VTENTRY`?

**Review by philipc — CHANGES_REQUESTED** [review]
Please add `EM_LOONGARCH` to `FLAGS_EM`.

File: crates/examples/src/readobj/elf.rs (hunk 1 above)
Comment (zhaixiaojuan) [reply crates/examples/src/readobj/elf.rs:3003]: Yes, I have added these two constants.

**Comment by philipc** [comment]
> Please add `EM_LOONGARCH` to `FLAGS_EM`.

This still needs fixing.

**Review by philipc — APPROVED** [review]

**Pull request merged**